Spec/RealtimeClientConnection.swift

```diff
@@ -3361,6 +3361,117 @@ class RealtimeClientConnection: QuickSpec {
                     }
                 }
 
+                // RTN22
+                it("Ably can request that a connected client re-authenticates by sending the client an AUTH ProtocolMessage") {
+                    let options = AblyTests.commonAppSetup()
+                    options.autoConnect = false
+                    options.token = getTestToken(ttl: 5.0)
```

Comment:
Well, OK then. I'd say you can just mock the AUTH message from the server, but well, I guess we want the real thing.